client/spec/authoring_spec.js

```diff
@@ -102,4 +102,30 @@ describe('authoring', function() {
         });
 
     });
+
+    describe('subnavigation menu', function () {
+        it('allows to create a new empty package', function () {
+            // 1. Navigate to the authoring main page.
+            //
+            // 2. Open the "plus icon" menu in subnav bar and click the
+            //    "Empty Package" link in the menu.
+            //
+            // 3. Get the newly created item's info box and check if it
+            //    contains an icon indicating that the item's type is indeed
+            //    "composite".
+            //
+            authoring.openAuthoringHome();
+
+            authoring.navbarMenuBtn.click();
+
+            authoring.newEmptyPackageLink.click().then(function () {
+                return authoring.infoIconsBox.all(
+                    by.css('.filetype-icon-composite')
```

Comment:
The basic idea is to move all selections on PageObjects. You can create a method that returns the list of elements filtered by the following css filter:  ".filetype-icon-composite".
And yes, some tests don't respect PageObjects pattern. We need to update them, but for sure those tests are not a good example to follow.